src/Authentication.gs

```diff
@@ -1,3 +1,6 @@
+/* Circuit breaking constant*/
+var backendDissabled = true;
```

Comment:
How will you use this variable? Because now It will be always true and it means that back-end is always disabled.